packages/core/src/components/DropdownNew/Dropdown.types.ts

```diff
@@ -85,6 +89,14 @@ export interface BaseDropdownProps<Item extends BaseListItemData<Record<string,
    * If true, the dropdown will have a clear button.
    */
   clearable?: boolean;
+  /**
+   * If true, the dropdown allows multiple selections.
+   */
+  multi?: boolean;
+  /*
+   * If true, the dropdown allows multiple lines of selected items.
+   */
+  multiline?: boolean;
```

Comment:
is multiline only available in multi mode?
if so, we should have an improvement task (not critical for now) to only allow passing one when the other is true

Reply:
talked about it - 2nd phase?